composer/loggers/wandb_logger.py

```diff
@@ -127,6 +129,26 @@ def log_file_artifact(self, state: State, log_level: LogLevel, artifact_name: st
             artifact.add_file(os.path.abspath(file_path))
             wandb.log_artifact(artifact, aliases=aliases)
 
+    def get_file_artifact(
+        self,
+        artifact_name: str,
+        destination: str,
+        chunk_size: int = 2**20,
+        progress_bar: bool = True,
+    ):
+        # Note: Wandb doesn't support progress bars for downloading
+        del chunk_size, progress_bar
+        import wandb
+
+        artifact = wandb.use_artifact(artifact_name)
+        with tempfile.TemporaryDirectory() as tmpdir:
+            artifact_folder = os.path.join(tmpdir, "artifact_folder")
+            artifact.download(root=artifact_folder)
+            # We only log one file per artifact
```

Comment:
Can you raise a `RuntimeError` here if this assumption does not hold? (In case if the user uploaded an artifact from a non-composer run?)